apps/etterlatte-trygdetid/src/main/kotlin/trygdetid/regler/BeregnTrygdetid.kt

```diff
@@ -10,40 +10,59 @@ import no.nav.etterlatte.libs.regler.finnFaktumIGrunnlag
 import no.nav.etterlatte.libs.regler.med
 import no.nav.etterlatte.libs.regler.og
 import java.time.LocalDate
+import java.time.Month
+import java.time.MonthDay
 import java.time.Period
 
 // TODO dato settes riktig senere
 val TRYGDETID_DATO: LocalDate = LocalDate.of(1900, 1, 1)
 
+data class TrygdetidPeriodMedPoengAar(
+    val fra: LocalDate,
+    val til: LocalDate,
+    val poengInnAar: Boolean,
+    val poengUtAar: Boolean
+)
+
 data class TrygdetidPeriodeGrunnlag(
-    val periodeFra: FaktumNode<LocalDate>,
-    val periodeTil: FaktumNode<LocalDate>
+    val periode: FaktumNode<TrygdetidPeriodMedPoengAar>
 )
 
+private fun TrygdetidPeriodMedPoengAar.erEnkelPoengAar() = fra.year == til.year && (poengInnAar || poengUtAar)
```

Comment:
navnet `erEnkelPoengAar` klinger ikke veldig godt hos meg 🤔 
Har ikke så mange gode forslag forsåvidt, kanskje frittstående poengår, evt. `erEttPoengAar`?

Reply:
Slett litt - tanken jeg hadde var "isSingle" men - single and simple begge ser ut til å være "enkel"

Reply:
Kanskje Ett har riktig betydning. Går med den fram til noen andre klager eller kommer med bedre :)